cpp/src/Slice/Parser.cpp

```diff
@@ -2695,6 +2693,24 @@ Slice::ClassDef::compactId() const
     return _compactId;
 }
 
+MetadataList
```

Comment:
The metadata API on the `Def` types just always forward to their corresponding `Decl`.